Sharkitty/OpenPype
# Bugfix: Newly downloaded workfile

## Changelog Description
Fix newly downloaded workfile (via `pre_copy_last_published_workfile` hook) wrongly marked as out of date.

## Testing notes:
- Download a workfile from the launcher (`_old` technique or locally unavailable workfile).
- It should be marked as up to date.
- Open a locally available workfile which is known to be marked as out of date.
- It should remain out of date.
